### ISSUE TITLE
nearleyc in a browser

### ISSUE THREAD

**Issue by tjvr** [Hardmath123/nearley#267]
As discussed elsewhere, we should have an API for invoking `nearleyc` from a script and/or in the browser. Then we can drop `docs/using-in-frontend.md` entirely.

Ideally, we would do this without rearranging the files in the repo, or breaking backwards compat in other ways.

**Comment by Hardmath123** [comment]
Yeah. It already works in our web interface in /www, so we just need some kind of wrapper?

**Comment by tjvr** [comment]
> some kind of wrapper

Pretty much! I propose:

* a `nearleyc` wrapper in `./nearleyc.js`, which can be imported via `require('nearley/nearleyc')`
* Making sure everything in `lib/` is actually UMD
* a nice API in the aforementioned `nearleyc` file (does it use streams? do we provide a `require-from-string` helper?)
